File: src/main/java/com/teamwizardry/wizardry/common/block/access/Invokers.java
package com.teamwizardry.wizardry.common.block.access;

import net.minecraft.block.BlockState;

/*****************
 * TEMPORARY CLASS
 *****************
 * Should be using invoker mixins (icky) or liblib foundation
 */
public class Invokers {
    public static class DoorBlock extends net.minecraft.block.DoorBlock { public DoorBlock(Settings settings) { super(settings); } }
    public static class StairsBlock extends net.minecraft.block.StairsBlock { public StairsBlock(BlockState baseBlockState, Settings settings) { super(baseBlockState, settings); } }
}

File: src/main/java/com/teamwizardry/wizardry/common/block/access/Invokers.kt
package com.teamwizardry.wizardry.common.block.access

import net.minecraft.block.AbstractBlock
import net.minecraft.block.BlockState
import net.minecraft.block.DoorBlock
import net.minecraft.block.StairsBlock

/*****************
 * TEMPORARY CLASS
 *
 * Should be using invoker mixins (icky) or liblib foundation
 */
class Invokers {
    class DoorBlock(settings: AbstractBlock.Settings?) : net.minecraft.block.DoorBlock(settings)
    class StairsBlock(baseBlockState: BlockState?, settings: AbstractBlock.Settings?) : net.minecraft.block.StairsBlock(baseBlockState, settings)
}